packages/flutter/lib/src/material/selection_area.dart

```diff
@@ -87,6 +88,9 @@ class SelectionArea extends StatefulWidget {
   /// Called when the selected content changes.
   final ValueChanged<SelectedContent?>? onSelectionChanged;
 
+  /// The separator to use when concatenating the selected text.
```

Comment:
I would explain the default value of empty string here, too, and then say that you might want to use `'\n'` instead.

Actually, maybe we should default to newline?

Reply:
@justinmc I don't think we should default to newline as that could introduce a breaking change to apps already using it and expecting no separator.

Reply:
Are there any existing apps that are expecting no separator, or would they consider that a bug?  I would guess that most expect as newline separator, but it's true I'm just guessing.  If you @dsyrstad have an app that expects no separator then that's pretty strong evidence we should keep that default.

Reply:
@justinmc I don't have an app which expects no separator, but I was working on an e2e test that was going to check what was copied from the clipboard (which is how I ran into this issue). If there are existing e2e tests out there expecting it as it is, I'd guess they'd break. Also, it may be that _users_ of apps historically expect it in a certain format, so you might break the users. I'm of the opinion to do the least harm possible. If an app wants a separator, it can configure it.

Reply:
True, thanks for sharing your context here!  I'm happy either way.

Reply:
I updated the doc comment. Let me know if you have any thoughts.